test/NNPDE_tests.jl

```diff
@@ -0,0 +1,123 @@
+using Flux
+using DiffEqFlux
+using ModelingToolkit
+using DiffEqBase, DiffEqOperators
+println("NNPDE_tests")
+using Test, NeuralNetDiffEq
+
+## Example 1, 1D ode
+@parameters t θ
+@variables u(..)
+@derivatives Dt'~t
+
+# 1D ODE and boundary conditions
+eq = Dt(u(t,θ)) ~ t^3 + 2*t + (t^2)*((1+3*(t^2))/(1+t+(t^3))) - u(t,θ)*(t + ((1+3*(t^2))/(1+t+t^3)))
+bcs = [u(0.) ~ 1.0 , u(1.) ~ 1.202]
+
+# Space and time domains
+domains = [t ∈ IntervalDomain(0.0,1.0)]
+
+# Method of lines discretization
+dx = 0.1
+order = 1
+discretization = MOLFiniteDifference(dx,order)
+
+# neural network and optimizer
+opt = Flux.ADAM(0.1)
+chain = FastChain(FastDense(1,12,Flux.σ),FastDense(12,1))
+
+pde_system = PDESystem(eq,bcs,domains,[t],[u])
+prob = NeuralNetDiffEq.NNPDEProblem(pde_system,discretization)
```

Comment:
```suggestion
prob = discretize(pde_system,discretization)
```

Reply:
It would actually be nice for this to kick out an OptimizationProblem, but that could come later.